docs/decisions/0002-forum_v1_migration_to_v2.rst

```diff
@@ -10,13 +10,13 @@ This document outlines the migration process from Forum v1 (written in Ruby) to
 Migration Decisions
 ===================
 
-1. Notifications Model
+Notifications Model
 ^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
this will fail the docs build
^ these numbers should be align with the headings like
```
Notifications Model
^^^^^^^^^^^^^^^^^^^
```
same comment for all headings.